src/python/sdk/pyproject.toml

```diff
@@ -10,6 +10,7 @@ packages = [{ include = "econia_sdk" }]
 aptos-sdk = "^0.5.1"
 python = "^3.8"
 autoflake = "^2.2.0"
+pdoc = "^14.0.0"
```

Comment:
Use Poetry dependency groups:
<https://python-poetry.org/docs/managing-dependencies/>

e.g. `poetry add --group docs pdoc`

Reply:
@bogberry addressed in bea4ab5